### PR TITLE
Fix #1893 Update load config

### LINKED ISSUE #1893: load_kube_config() got an unexpected key word argument 'kube_config_path'
When using load_config() and "kube_config_path" in kwargs:
https://github.com/kubernetes-client/python/blob/16935036252be0fdf164f8ce624bd71eb8d35e89/kubernetes/base/config/__init__.py#L24
load_config(kube_config_path="/")

Received this error:
load_kube_config() got an unexpected key word argument 'kube_config_path'.

because:
image

image

### PR DESCRIPTION
**Pull request by yabea**

### CODE REVIEW

**Comment by yliaog** [comment]
https://github.com/kubernetes-client/python/issues/1893

File: kubernetes/base/config/__init__.py [hunk 1]
```diff
@@ -33,8 +33,10 @@ def load_config(**kwargs):
...
     load_incluster_config functions.
     """
-    if "kube_config_path" in kwargs.keys() or exists(expanduser(KUBE_CONFIG_DEFAULT_LOCATION)):
+    if "config_file" in kwargs or exists(expanduser(KUBE_CONFIG_DEFAULT_LOCATION)):
         load_kube_config(**kwargs)
+    elif "config_dict" in kwargs:
+        load_kube_config_from_dict(**kwargs)
     else:
         print(
             "kube_config_path not provided and "
```
Comment (yliaog) [inline kubernetes/base/config/__init__.py:42]: could you fix the above line: kube_config_path

**Comment by blackstorm** [comment]
Any update here?

File: kubernetes/base/config/__init__.py (hunk 1 above)
Comment (yabea) [reply kubernetes/base/config/__init__.py:42]: which line?

**Pull request closed without merging**